mlos_bench/mlos_bench/services/remote/azure/azure_auth.py

```diff
@@ -24,6 +27,7 @@ class AzureAuthService(Service, SupportsAuth):
     """
 
     _REQ_INTERVAL = 300   # = 5 min
+    _TENTANT_ID = "72f988bf-86f1-41af-91ab-2d7cd011db47"
```

Comment:
maybe we should not hardcode it. let's move it to the external config outside of git

Reply:
Absolutely - this should not be in the repo. It should be in the globals_config or else can we look it up from subscription (since that is already required I think).

Reply:
Good point! I will remove this, and I realize the `generate-azure-credentials-config.*` scripts already include the tenant into the global config.

Reply:
Removed the hardcoded value! Theres a placeholder for it now, and it should come from the global config.